### DISCUSSION TITLE
Is there a way to declare tab_bar styles in .toml colour files?

### DISCUSSION THREAD

**Discussion by lukamanitta** [wezterm/wezterm#1219] [Q&A]
I'm trying to port a number of colour schemes over to Wezterm, including some custom tab bar styles, but the inbuilt style examples don't demonstrate this customisation, and I can't seem to figure out how to do this / if it is possible.

**Comment by wez** [comment]
No, sorry!  Do you have example(s) of color schemes that include tab bar coloring?

**Reply by lukamanitta** [reply]
I'm porting the neovim colourschemes from [Catppuccino.nvim](https://github.com/Pocco81/Catppuccino.nvim) to Wezterm, and defining my own tab bar styles as part of this. Because there are multiple schemes, I'd like to be able to utilise either the `.toml` method, or the custom `color_schemes` method (as per [this part of the config](https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-your-weztermlua)), instead of simply copying the entire style over the `colors` field. However, the `color_schemes` method does not seem to style the tab bar either, and I'm not sure if it's an error on my part or if it is simply not supported.
